lib/Conversion/TorchToMhlo/PopulatePattern.h

```diff
@@ -0,0 +1,22 @@
+//===------------------------------------------------------------*- C++ -*-===//
```

Comment:
nit: rename file to PopulatePatterns.h for consistency with https://github.com/llvm/torch-mlir/blob/main/lib/Conversion/TorchToLinalg/PopulatePatterns.h